tools/topology/m4/bytecontrol.m4

```diff
@@ -7,19 +7,19 @@ define(`CONTROLBYTES_MAX',
 `#$1'
 `	max STR($2)')
 
-dnl CONTROLMIXER_TLV(comment, value)
+dnl CONTROLBYTES_TLV(comment, value)
```

Comment:
Can you say in the commit message what this fixes, it's not clear why you are making this change ?

Reply:
it seems pretty clear, just look at the next line:
```
define(`CONTROLBYTES_TLV',
```
it's a pure copy-paste comment correction, and the file name is also "bytecontrol.m4" as opposed to "mixercontrol.m4" where that copying comes from. But yes, I can add that to the commit message...